tests/pytests/test_http_monitor.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+import zlib
+import json
+from datetime import datetime
+try:
+    import urllib.request as urllib_req
+except ImportError:
+    import urllib2 as urllib_req
+from statistics_checker import MonitorStatsChecker
+from statistics_checker import categories_combination
+
+import elliptics
+
+path_categories = {"/cache": elliptics.core.monitor_stat_categories.cache,
+                   "/io": elliptics.core.monitor_stat_categories.io,
+                   "/commands": elliptics.core.monitor_stat_categories.commands,
+                   "/backend": elliptics.core.monitor_stat_categories.backend,
+                   "/stats": elliptics.core.monitor_stat_categories.stats,
+                   "/procfs": elliptics.core.monitor_stat_categories.procfs,
+                   "/top": elliptics.core.monitor_stat_categories.top,
+                   "/all": elliptics.core.monitor_stat_categories.all}
+
+
+@pytest.fixture(scope="session")
+def address(servers):
+    return servers.remotes[0].split(':')[0] + ':' + servers.monitors[0]
+
+
+class TestHttpMonitor:
+
+    @pytest.mark.parametrize("path", path_categories.keys())
+    @pytest.mark.usefixtures("servers")
+    def test_http_statistics_by_path(self, address, routes, backends_combination, path):
+        '''Requests all possible combination of path and backends one by one and checks statistics'''
+
+        start = datetime.now()
+        json_stat = self.__get_statistics_by_path(address=address,
+                                                  path=path,
+                                                  backends=backends_combination)
+        end = datetime.now()
+
+        checker = MonitorStatsChecker(json_statistics=json_stat,
+                                      time_period=(start, end),
+                                      categories=path_categories[path],
+                                      address=routes.addresses()[0],
+                                      routes=routes,
+                                      backends_combination=backends_combination)
+        checker.check_json_stat()
+
+    @pytest.mark.parametrize("categories", categories_combination())
+    @pytest.mark.usefixtures("servers")
+    def test_http_statistics_by_category(self, address, routes, backends_combination, categories):
+        '''Requests all possible combination of categories and backends one by one and checks statistics'''
+
+        start = datetime.now()
+        json_stat = self.__get_statistics_by_categories(address=address,
+                                                        categories=categories,
+                                                        backends=backends_combination)
+        end = datetime.now()
+
+        checker = MonitorStatsChecker(json_statistics=json_stat,
+                                      time_period=(start, end),
+                                      categories=categories,
+                                      address=routes.addresses()[0],
```

Comment:
There is no guarantee that `address` and `routes.addresses()[0]` is about the same node here